bin/hooks/pre-commit

```diff
@@ -14,9 +14,8 @@ fi
 # Run PHP_CodeSniffer
 echo "Running PHP_CodeSniffer..."
 
-git diff --name-only --cached --diff-filter=ACMRTUXB | xargs vendor/bin/phpcs $standard
-if [ $? != 0 ]
-then
+composer run lint:plugin
```

Comment:
Where is this command coming from? While it's handy to have a Composer script to run, I don't see a definition for this anywhere (nor is WP Enforcer only used for plugins, as this command seems to indicate).